vector_icons/leo_overrides.json

```diff
@@ -0,0 +1,27 @@
+{
+  "//components/vector_icons/reload.icon": "leo_browser_refresh",
+  "//components/vector_icons/not_secure_warning.icon": "leo_warning_triangle_filled",
+  "//components/vector_icons/https_valid.icon": "leo_lock_plain",
+  "//components/vector_icons/forward_arrow.icon": "leo_browser_forward",
+  "//components/vector_icons/extension.icon": "leo_browser_extensions",
+  "//components/vector_icons/back_arrow.icon": "leo_browser_back",
+  "//components/omnibox/browser/vector_icons/star.icon": "leo_browser_bookmark_normal",
+  "//components/omnibox/browser/vector_icons/star_active.icon": "leo_bookmark_added",
+  "//components/omnibox/browser/vector_icons/share.icon": "leo_share",
+  "//components/omnibox/browser/vector_icons/share_win.icon": "leo_share_windows",
+  "//components/omnibox/browser/vector_icons/share_mac.icon": "leo_share_macos",
+  "//components/omnibox/browser/vector_icons/install_desktop.icon": "leo_pwa_install",
+  "//components/omnibox/browser/vector_icons/http.icon": "leo_warning_circle_filled",
+  "//components/omnibox/browser/vector_icons/find_in_page.icon": "leo_window_search",
+  "//components/omnibox/browser/vector_icons/bookmark.icon": "leo_browser_bookmark_normal",
+  "//chrome/app/vector_icons/zoom_plus.icon": "leo_search_zoom_in",
+  "//chrome/app/vector_icons/zoom_minus.icon": "leo_search_zoom_out",
+  "//chrome/app/vector_icons/navigate_stop.icon": "leo_close",
+  "//chrome/app/vector_icons/navigate_stop_windows.icon": "leo_close",
+  "//chrome/app/vector_icons/navigate_home.icon": "leo_browser_home",
+  "//chrome/app/vector_icons/media_toolbar_button.icon": "leo_music_tones",
+  "//chrome/app/vector_icons/download_toolbar_button.icon": "leo_download",
+  "//chrome/app/vector_icons/download_in_progress.icon": "leo_download",
+  "//chrome/app/vector_icons/browser_tools.icon": "leo_hamburger_menu",
+  "//chrome/app/vector_icons/battery_saver.icon": "leo_energy_saver"
```

Comment:
sort please?

Reply:
Whoops, accidentally sorted the wrong way in my IDE :facepalm: